learn/protocol/sync-protocol/nodes-and-sync.md

```diff
@@ -17,6 +17,27 @@ Light nodes rely on full or history nodes to sync through two mechanisms:
 - [Light Macro Sync](light-macro-sync.md)
 - [Block Live Sync](block-live-sync.md)
 
+## Pico nodes
+
+Pico nodes are a variant of light nodes designed to sync with the Nimiq network more efficiently by using a simplified consensus mechanism. Like light nodes, they do not download the full chain and do not participate in block production or validation.
+
+Pico nodes only sync macro blocks and rely on a trust-based consensus mechanism to identify the canonical chain. They do not verify ZKPs but instead establish consensus by querying multiple peers for the most recent election block. If the responses are consistent across peers, the pico node assumes this block is canonical and transitions to block live sync.
+
+Pico nodes rely on full or history nodes to sync through two mechanisms:
+
+- [Pico Macro Sync](pico-macro-sync.md)
+- [Block Live Sync](block-live-sync.md)
+
+The table below compares light and pico nodes:
```

Comment:
Do you think makes sense to add a new column with also Full Nodes and move the table to its own section with unique h2 (`## Sync comparisson table`), so we can link the table directly.

Also I think if we do that we could add a new row called `Web Client` where light and pico are supported but not full node. What do you think?